### PR TITLE
Loki 1.2 update - playground release

### PR DESCRIPTION
**Pull request by piontec**
Towards https://github.com/giantswarm/giantswarm/issues/7746

- Official release stripped down to loki+promtail only.
- Playground quality met for loki 1.2.0 release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,28 @@
 
 # loki-stack-app chart
 
-Giant Swarm offers a loki-stack Managed App which can be installed in tenant clusters.
-Here we define the loki chart with its templates and default configuration.
+Giant Swarm offers a loki-stack Managed App which can be installed in Kubernetes clusters.
```
Comment (puja108) [inline README.md]: I would not call it "Managed" and why even mention Giant Swarm here? Just describe what it is: a chart that installs a minimal loki stack incl promtail and loki

File: helm/loki-stack-app/charts/loki/values.yaml [hunk 2]
```diff
@@ -67,7 +67,7 @@ config:
 
 image:
   repository: grafana/loki
```
Comment (puja108) [inline helm/loki-stack-app/charts/loki/values.yaml]: maybe you wanna also add a default registry like in https://github.com/giantswarm/coredns-app/blob/master/helm/coredns-app/values.yaml#L23

File: helm/loki-stack-app/charts/promtail/values.yaml [hunk 3]
```diff
@@ -8,7 +8,7 @@ deploymentStrategy: RollingUpdate
 
 image:
   repository: grafana/promtail
```
Comment (puja108) [inline helm/loki-stack-app/charts/promtail/values.yaml]: here also add registry default

File: helm/loki-stack-app/charts/loki/values.yaml (hunk 2 above)
Comment (puja108) [reply helm/loki-stack-app/charts/loki/values.yaml]: ah, just saw you template registry on the higher level, disregard here and below

File: README.md (hunk 1 above)
Comment (piontec) [reply README.md]: OK, this decription comes from "pre-agreement" catalogs layout. I think we should avoid "managed" and "giant swarm" in playground in general

**Pull request merged**